openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2266,7 +2266,11 @@ The following is a non-normative example of a Credential Request with Credential
 
 ### Credential Response
 
-The value of the `credential` claim in the Credential Response MUST be a string that is the base64url-encoded representation of the issued Credential.
+The value of the `credential` claim in the Credential Response MUST be a string that is the base64url-encoded representation of the CBOR-encoded `IssuerSigned` structure, as defined in [@!ISO.18013-5].
```

Comment:
Do you think we need to say anything about the content of `IssuerSigned`? In the `IssuerSigned` struct, the `nameSpaces` element is optional. IMO it might be worth saying something like:

> This structure (SHOULD/MUST) contain all `Namespaces` and `IssuerSignedItems` that are included in the `AuthorizedNamespaces` of the `MobileSecurityObject`.

Reply:
i think this makes sense

Reply:
```suggestion
The value of the `credential` claim in the Credential Response MUST be a string that is the base64url-encoded representation of the CBOR-encoded `IssuerSigned` structure, as defined in [@!ISO.18013-5]. This structure SHOULD contain all `Namespaces` and `IssuerSignedItems` that are included in the `AuthorizedNamespaces` of the `MobileSecurityObject`.
```

Reply:
@awoie could you please accept the suggestion, so that we can merge a PR?